packages/models/test/fixtures/runner.mock.ts

```diff
@@ -0,0 +1,13 @@
+import { join } from 'node:path';
+import { AuditReport, RunnerConfig } from '../../src';
+
+export function runnerConfig(
+  audits: AuditReport[],
+  outputFile = join('tmp', `out.${Date.now()}.json`),
+): RunnerConfig {
+  return {
+    command: 'echo',
+    args: [`${JSON.stringify(audits)} > ${outputFile}`],
+    outputFile,
+  };
+}
```

Comment:
I don't think this is being used, is it? Looks like a duplicate of `echoRunnerConfig` without the platform-dependant JSON handling.

The default `outputFile` parameter of `join('tmp', `out.${Date.now()}.json`)` could be useful, though :thinking: Maybe you can add it to `echoRunnerConfig` and remove this function?

Reply:
@Tlacenka should have moved her echo fn in here of just adopt the code.

I will refactor in in another PR

Reply:
This function was replaced by echoRunnerConfig in my Windows PR, as far as I remember.

Reply:
The reason why I created a new name for the function was that `runnerConfig` does not tell me what command or function the configuration serves. I still see no reason to have the duplicate here when the default value could have just been inserted to the `echoRunnerConfig` function.

Reply:
> This function was replaced by echoRunnerConfig in my Windows PR, as far as I remember.
It got not renamed ... will merge them in 1 file